### PR TITLE
Leah style css

### PR DESCRIPTION
**Pull request by Hunter-aw**
i did cool style stuff

### CODE REVIEW

File: dist/style.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+body{
+    margin:0;
+    font-family: Arial, Helvetica, sans-serif;
+    background-color:lightblue;
+}
+
+div{
+    color:darkred;
+}
```
Comment (Hunter-aw) [inline dist/style.css:9]: not crazy about this color

Comment (LeahChercham) [reply dist/style.css:9]: Darkred is super cool !

Comment (Tomshsh) [reply dist/style.css:9]: im a teapot

File: dist/style.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+body{
+    margin:0;
```
Comment (simonabali) [inline dist/style.css:2]: Add some more space, too crowded :)

File: dist/style.css (hunk 1 above)
Comment (fnando-gomez) [reply dist/style.css:9]: Nice idea to have a dark-theme

Comment (jovandyaz) [reply dist/style.css:9]: Órale pues, dale!

File: dist/style.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+body{
```
Comment (simonabali) [inline dist/style.css:1]: Happy with this

**Comment by jovandyaz** [comment]
Ajá...y luego? @Tomshsh

**Pull request merged**